swagger/v3/component_schemas/NPQParticipantAttributes.yml

```diff
@@ -24,3 +25,8 @@ properties:
     type: array
     items:
       $ref: "#/components/schemas/NPQEnrolment"
+  participant_id_changes:
```

Comment:
just to confirm (as can't see in review app yet) that this will return an empty array as well?